Dockerfile.prow

```diff
@@ -1,4 +1,4 @@
-FROM registry.ci.openshift.org/stolostron/builder:go1.18-linux as builder
+FROM registry.ci.openshift.org/stolostron/builder:go1.20-linux as builder
```

Comment:
I have a feeling the same issue with the other two CLC repos will happen here. I recommend just upgrading to 1.19.

Reply:
Yep @lennysgarage you also ran into the same issue?

Reply:
Yeah same issue here, https://github.com/stolostron/provider-credential-controller/pull/64